sys/shell/commands/sc_sx127x.c

```diff
@@ -0,0 +1,586 @@
+/*
```

Comment:
I tried but it was becoming a mess with other #ifdef in this file. I thought splitting the 2 would make the code more readable. Maybe we can factorize the interface list and config commands.